grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package network;
+
+option go_package = "smatov/dataplane";
+
+service NetworkService {
+    rpc AddVPC(AddVPCRequest) returns (VPCResponse);
+    rpc UpdateVPC(UpdateVPCRequest) returns (VPCResponse);
+    rpc DeleteVPC(DeleteVPCRequest) returns (DeleteResponse);
+    rpc ListVPCs(ListVPCsRequest) returns (ListVPCsResponse);
+
+    rpc AddPeeringInterface(AddPIFRequest) returns (PIFResponse);
+    rpc UpdatePeeringInterface(UpdatePIFRequest) returns (PIFResponse);
+    rpc DeletePeeringInterface(DeletePIFRequest) returns (DeleteResponse);
+    rpc ListPeeringInterfaces(ListPIFsRequest) returns (ListPIFsResponse);
+
+    rpc AddPeeringPolicy(AddPeeringPolicyRequest) returns (PeeringPolicyResponse);
+    rpc UpdatePeeringPolicy(UpdatePeeringPolicyRequest) returns (PeeringPolicyResponse);
+    rpc DeletePeeringPolicy(DeletePeeringPolicyRequest) returns (DeleteResponse);
+    rpc ListPeeringPolicies(ListPeeringPoliciesRequest) returns (ListPeeringPoliciesResponse);
+}
+
+message VPC {
+    string vpc_name = 1;
```

Comment:
Do VPCs need an ID? The string is easier to read for humans, but less efficient to compare + we need to ensure unicity somehow?